src/app/home-page/home-logged-out/home.component.ts

```diff
@@ -88,10 +87,6 @@ export class HomeComponent extends Base implements OnInit, AfterViewInit {
     this.homePageService.goToSearch(searchValue);
   }
 
-  onSelect(data: TabDirective): void {
```

Comment:
To be clear, this is unrelated to the crux of your PR, right? Is it just a dangling reference to ngx-bootstrap that wasn't being used?

Reply:
Yes it is just a dangling unused reference to ngx-bootstrap. I figured that since it was a very easy fix, I would just include it into this PR.